translations/pl.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Confirmed": "Zachorowania",
+    "New Cases": "Nowe (Dziś)",
+    "Deaths": "Zgony",
+    "New Deaths": "Nowe (Dziś)",
+    "Serious" : "W Stanie Ciężkim",
+    "Recovered": "Wyleczenia",
+    "Active": "Choruje",
+    "Cases Per M" : "Zachorowania Na Milion",
+    "Worldwide" : "Świat",
+    "statistic taken at ": "ostatnia aktualizacja ",
+    "N/A" : "N/A"
+}
```

Comment:
@pawelros 
we will need here, the countries's translations

Reply:
I based my translation of en, was looking at how to translate the countries as well. Sure thing, will provide that too.

Reply:
Done